usr/lib/tik/modules/pre/10-welcome

```diff
@@ -3,34 +3,43 @@
 # SPDX-FileCopyrightText: Copyright 2024 Richard Brown
 # SPDX-FileCopyrightText: Copyright 2024 Raymond Yip
 
+displayWarning=false
```

Comment:
How about a more specific variable name.

Reply:
Fixed on the next commit